File: JoKenPoh.py
from time import sleep
from random import randint
while True:
    computador = randint(0,2)
    itens = ["PEDRA", "PAPEL", "TESOURA"]
    print("Suas Opcoes: ")
    print("[0] PEDRA")
    print("[1] PAPEL")
    print("[2] TESOURA")
    jogador = int(input("Qual a sua jogada: "))
    print("JO")
    sleep(1)
    print("KEN")
    sleep(1)
    print("POHH!!")
    print("-=-" * 15)
    print(f"computador jogou {itens[computador]} ".upper())
    print(f"Voce jogou {itens[jogador]} ".upper())
    print("-=-" * 15)
    if(computador == 0):
        if(jogador == 0):
            print("EMPATE!!!")
        elif(jogador == 1):
            print("JOGADOR GANHOU!!!")
        elif(jogador == 2):
            print("COMPUTADOR GANHOU!!!")
        else:
            print("JOGADA INVALIDA!!!")
    elif(computador == 1):
        if(jogador == 0):
            print("COMPUTADOR GANHOU!!!")
        elif(jogador == 1):
            print("EMPATE!!!")
        elif(jogador == 2):
            print("JOGADOR GANHOU!!!")
        else:
            print("JOGADA INVALIDA!!!")
    elif(computador == 2):
        if(jogador == 0):
            print("JOGADOR GANHOU!!!")
        elif(jogador == 1):
            print("JCOMPUTADOR GANHOU!!!")
        elif(jogador == 2):
            print("EMPATE!!!")
        else:
            print("JOGADA INVALIDA!!!")
    print()
    op = str(input('Quer jogar novamente? '))
    if op in 'Nn':
        break

File: JoKenPoh.cpp
#include <time.h>
#include <iostream>
#include <windows.h>
using namespace std;
int main() {
    char novamente;
    int jogador, comp;
    novamente = 'S';
    while (novamente == 's' || novamente == 'S') {
    srand(time(NULL));
    comp = rand() % 2;
    cout << "[0] Para PEDRA" << endl;
    cout << "[1] Para PAPEL" << endl;
    cout << "[2] Para TESOURA" << endl;
    cout << "Faca sua escolha: ";
    cin >> jogador;
    Sleep(500);
    cout << "JO" << endl;
    Sleep(500);
    cout << "KEN" << endl;
    Sleep(500);
    cout << "POH!!!" << endl;
    if(jogador == 0) {
        if(comp == 0) {
            cout << "Computador jogou: " << comp << endl;
            cout << "EMPATE "<< endl;
         }
        if(comp == 1) {
            cout << "Computador jogou: " << comp << endl;
            cout << "COMPUTADOR GANHOU" << endl;
        }
        if(comp == 2) {
            cout << "Computador jogou: " << comp << endl;
            cout << "JOGADOR GANHOU" << endl;
        }
    }
    if(jogador == 1) {
        if(comp == 0) {
            cout << "Computador jogou: " << comp << endl;
            cout << "JOGADOR GANHOU" << endl;
        }
        if(comp == 1) {
            cout << "Computador jogou: " << comp << endl;
            cout << "EMPATE" << endl;
        }
        if(comp == 2) {
            cout << "Computador jogou: " << comp << endl;
            cout << "COMPUTADOR GANHOU" << endl;
        }
    }
    if(jogador == 2) {
        if(comp == 0) {
            cout << "Computador jogou: " << comp << endl;
            cout << "COMPUTADOR GANHOU" << endl;
        }
        if(comp == 1) {
            cout << "Computador jogou: " << comp << endl;
            cout << "JOGADOR GANHOU" << endl;
        }
        if(comp == 2) {
            cout << "Computador jogou: " << comp << endl;
            cout << "EMPATE" << endl;
        }
    }
    cout << "OBRIGADO POR JOGAR" << endl;
    cout << "Quer jogar novamente???" << endl;
    cin >> novamente;
    }
}
